draft-ietf-oauth-jwk-thumbprint-uri.xml

```diff
@@ -108,11 +108,13 @@
       </t>
     </section>
 
-      <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
-  <t>
-  Hash algorithm identifiers used in JWK Thumbprint URIs are strings
-  registered in the IANA "Named Information Hash Algorithm Registry" <xref target="IANA.Hash.Algorithms"/>.
-  </t>
+    <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
+      <t>
+	Hash algorithm identifiers used in JWK Thumbprint URIs MUST be values from the "Hash Name String" column
+	in the IANA "Named Information Hash Algorithm" registry <xref target="IANA.Hash.Algorithms"/>.
+	JWK Thumbprint URIs with hash algorithm identifiers not found in this registry are considered invalid
+	and applications using these thumbprints will need appropriate error handling mechanisms.
```

Comment:
we don't need to define `appropriate error handling mechanisms` in this document itself?

Reply:
The appropriate error handling mechanisms will be context-specific.  That said, I can make the error handling language less suggestive and just say that applications will need to handle the error.  "Appropriate" is overkill.  Good catch.